core/src/components/chip/chip.scss

```diff
@@ -8,24 +8,23 @@
   --background: #{rgba($text-color-rgb, .12)};
   --color: #{rgba($text-color-rgb, .87)};
 
-  @include border-radius(16px);
+  @include border-radius(calc(16 / 14 * 1em));
```

Comment:
With this code, the corners aren't fully rounded if the text is wrapping.

Should we instead do e.g. a hack of 9999px to make sure all four corners round equally and the short corners are fully rounded? I didn't see anywhere in the code that we're currently doing that, so I wasn't sure what we want to do here.

with current code:
<img width="269" alt="Screenshot 2023-07-25 at 4 15 18 PM" src="https://github.com/ionic-team/ionic-framework/assets/14926794/2edc167a-b199-43c5-97e0-d4b38c16fb51">

with 9999px:
<img width="270" alt="Screenshot 2023-07-25 at 4 10 38 PM" src="https://github.com/ionic-team/ionic-framework/assets/14926794/f4ad616e-69ec-4c46-ae5c-d48c73ad11b2">

Reply:
Why not use `1rem` instead? `@include border-radius(1rem);`

Reply:
The border radius values should not change as the text scales -- we only want the text to change.

Reply:
if i set it to `1rem`, the ios ones look like this at small sizes

<img width="240" alt="Screenshot 2023-07-27 at 9 36 43 AM" src="https://github.com/ionic-team/ionic-framework/assets/14926794/36b3b7ac-4bd0-4509-8110-2bfdfd3b925f">

i assume we want the corners to be fully rounded, not like this.

this is happening because of the requirements in the ticket:

1. the ios font size has a max & min value
2. height is relative to that font size (and therefore also clamped)

therefore, `1rem` can be much smaller than `1em`, as seen here. so the border radius needs to be in `em` not in `rem` to adjust accordingly.

do we want to
a) change it to `1rem` or
b) have me add a comment?

Reply:
if we set it explicitly to 16px, it looks like this at large font sizes:

<img width="729" alt="Screenshot 2023-07-27 at 10 40 54 AM" src="https://github.com/ionic-team/ionic-framework/assets/14926794/ed6d7e75-2590-4bed-93a6-0d3cd4546b7f">